.ci/deps.sh

```diff
@@ -58,3 +58,10 @@ if [ ! -e ~/.local/bin/flawfinder ]; then
   sed -i '1s/.*/#!\/usr\/bin\/env python2/' ~/.local/bin/flawfinder
   chmod +x ~/.local/bin/flawfinder
 fi
+
+# Shards commands
+if [ ! -e ~/.local/bin/ameba ]; then
+  shards install
```

Comment:
Need to add a `ShardsRequirement` to package_manager.

But .. as you are using travis apt , and circle is disabled anyway, another option is to enhance DistributionRequirement to support custom repis.

Reply:
Enhancing `DistributionRequirement` is a better option as this is required for only one bear right now and I don't think `shards` will be required for any other bear in future also.

Reply:
@jayvdb, `Crystal` and `ameba` both can be installed using `brew`. 
Ref [crystal-lang using brew](https://crystal-lang.org/docs/installation/on_mac_osx_using_homebrew.html), [ameba using brew](https://github.com/veelenga/ameba#os-x).

Note: I have also written brew dependency in the bear.